src/docfx/lib/MarkdownUtility.cs

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Markdig;
+using Markdig.Extensions.Yaml;
+using Markdig.Parsers;
+using Markdig.Renderers;
+using Markdig.Syntax;
+using Markdig.Syntax.Inlines;
+using Microsoft.DocAsCode.MarkdigEngine.Extensions;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    /// <summary>
+    /// Converts markdown to html
+    /// </summary>
+    internal static class MarkdownUtility
+    {
+        // In docfx 2, a localized text is prepended to quotes beginning with
+        // [!NOTE], [!TIP], [!WARNING], [!IMPORTANT], [!CAUTION].
+        //
+        // Docfx 2 reads localized tokens from template repo. In docfx3, build (excluding static page generation)
+        // does not depend on template, thus these tokens are managed by us.
```

Comment:
Can we put loc token into some dependency or resoruce for better extensibility?

Reply:
I'd like to push these to the frontend. We should not care what the **visual** looks like. Build just add `class="warning"`, frontend can determine the localized visual, either a text or icons using css/js